ROMFS/CMakeLists.txt

```diff
@@ -105,7 +105,10 @@ endif()
 
 add_custom_command(OUTPUT romfs_extras.stamp
 	COMMAND ${CMAKE_COMMAND} -E make_directory ${romfs_gen_root_dir}/extras/
-	COMMAND ${CMAKE_COMMAND} -E copy_directory ${PX4_BINARY_DIR}/romfs_extras/ ${romfs_gen_root_dir}/extras/
+	COMMAND if [ -d  ${PX4_BINARY_DIR}/romfs_extras ]$<SEMICOLON> then
```

Comment:
This is kind of silly, but what if you just did `cmake -E make_directory ${PX4_BINARY_DIR}/romfs_extras` first? It's harmless if it already exists, and worst case we end up with an empty extras dir (also harmless).